shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Utxo.hs

```diff
@@ -134,8 +135,8 @@ data UtxoPredicateFailure era
       !Coin -- the minimum fee for this transaction
       !Coin -- the fee supplied in this transaction
   | ValueNotConservedUTxO
-      !(Core.Value era) -- the Coin consumed by this transaction
-      !(Core.Value era) -- the Coin produced by this transaction
+      !(Delta (Core.Value era)) -- the Coin consumed by this transaction
+      !(Delta (Core.Value era)) -- the Coin produced by this transaction
```

Comment:
@mrBliss - is it okay that I have changed the serialization for this predicate failure? I do not know who all uses the failure serializations.

Reply:
I would expect `forall (x :: Word64). toCBOR (word64ToCoin x) == toCBOR (toDelta (word64ToCoin x)` to be true, in which case this change is fine.

Reply:
ok, great! thank you for (quick) checking!